packages/thunderstore-api/src/index.ts

```diff
@@ -1,5 +1,5 @@
 export interface RequestConfig {
-  apiHost: string;
+  apiHost?: string;
```

Comment:
The diff doesn't make it obvious, so why do we want to change the host to be optional? Is some default value provided somewhere else (not visible in this diff)?